### PR TITLE
Ensures that OPDS For Distributors Monitor results are captured even …

### PR DESCRIPTION
**Pull request by dbernstein**
…when there are database

failures during import.

## Description

This update ensures that a timestamp record is saved even if there is a SQL failure during the import.

## Motivation and Context
https://ebce-lyrasis.atlassian.net/browse/PP-189

## How Has This Been Tested?
There is a unit test that verifies that a timestamp record is created even when a sql error occurs.

## Checklist

- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

File: core/monitor.py
Comment (RishiDiwanTT) [inline core/monitor.py:180]: Is this commit  required if we are forcing the timestamp object in the `except` block?

Comment (RishiDiwanTT) [reply core/monitor.py:180]: If there is a failure expected later on, do we want partial data saved?

**Review by RishiDiwanTT — COMMENTED** [review]
Just one query, looks good otherwise :+1:

File: core/monitor.py
Comment (dbernstein) [reply core/monitor.py:180]: That's a good point.  Good catch @RishiDiwanTT .  I'll remove the commit.

**Comment by dbernstein** [comment]
@RishiDiwanTT :  this one  should be good to go pending your approval.

**Review by RishiDiwanTT — APPROVED** [review]
:+1:

**Pull request merged**
